daemon/start_windows.go

```diff
@@ -21,8 +33,6 @@ func (daemon *Daemon) getLibcontainerdCreateOptions(container *container.Contain
 				config.KernelFile = v
 			case "lcow.initrd":
 				config.InitrdFile = v
-			case "lcow.vhdx":
```

Comment:
Is this breaking something?

Reply:
It's never been "supported" (bearing in mind lcow has to-date always been experimental from docker calling it).